test/JDBC/upgrade/14_5/preparation/BABEL-3556-vu-prepare.sql

```diff
@@ -0,0 +1,121 @@
+-- To test out of range values for fixeddecimal
```

Comment:
why we have added prepare files to each upgrade source directory, what is different for each release?

Reply:
There is behaviour differences in each release which means expected files will be different that's why I have to add  prepare files separately to each upgrade source directory.

Reply:
that is strange, you mean there is behavior difference across all releases or it starts to behave differently from one release onwards.

Reply:
In 13.4 when we try to do explicit cast from INT8 to fixeddecimal it is giving "data type fixeddecimal is not supported yet", Other than that there is no difference

Reply:
One trick could be to create 2 different test cases, e.g.,
- BABEL-3556
- BABEL-3556-13_4

Reference: https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/270/files